test/integration/targets/apt/tasks/main.yml

```diff
@@ -15,11 +15,29 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-- include: 'apt.yml'
```

Comment:
As mentioned on Slack, if you add your changes to the end of `main.yml` this will make the diff easier to read.